# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/common/event_tracer.h
# Commit: 13282d7a47a5
# Full Hash: 13282d7a47a57a73be8d060efd9645b4d642f03e
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 3064469c073d -r 13282d7a47a5 gfx/angle/checkout/src/common/event_tracer.h
--- a/gfx/angle/checkout/src/common/event_tracer.h	Sat Sep 07 13:36:46 2019 +0000
+++ b/gfx/angle/checkout/src/common/event_tracer.h	Sat Sep 07 13:38:36 2019 +0000
@@ -10,9 +10,9 @@
 
 namespace angle
 {
-
-const unsigned char *GetTraceCategoryEnabledFlag(const char *name);
-angle::TraceEventHandle AddTraceEvent(char phase,
+const unsigned char *GetTraceCategoryEnabledFlag(PlatformMethods *platform, const char *name);
+angle::TraceEventHandle AddTraceEvent(PlatformMethods *platform,
+                                      char phase,
                                       const unsigned char *categoryGroupEnabled,
                                       const char *name,
                                       unsigned long long id,